src/dynamics/se/dp_coupling.F90

```diff
@@ -547,20 +550,24 @@ subroutine derived_phys_dry(cam_runtime_opts, phys_state, phys_tend)
    ! Finally compute energy and water column integrals of the physics input state.
 
 !   use constituents,   only: qmin
-   use cam_ccpp_cap,   only: cam_constituents_array
-   use cam_constituents, only: const_get_index
-   use physics_types,  only: lagrangian_vertical
-   use physconst,      only: cpair, gravit, zvir, cappa
-   use cam_thermo,     only: cam_thermo_update
-   use physics_types,  only: rairv, zvirv
-   use shr_const_mod,  only: shr_const_rwv
-   use geopotential_t, only: geopotential_t_run
+   use ccpp_constituent_prop_mod, only: ccpp_constituent_prop_ptr_t
+   use cam_ccpp_cap,      only: cam_constituents_array
+   use cam_ccpp_cap,      only: cam_model_const_properties
+   use cam_constituents,  only: const_get_index
+   use physics_types,     only: lagrangian_vertical
+   use physconst,         only: cpair, gravit, zvir, cappa
+   use cam_thermo,        only: cam_thermo_update
+   use physics_types,     only: cpairv, rairv, zvirv
+   use physics_grid,      only: columns_on_task
+   use shr_const_mod,     only: shr_const_rwv
```

Comment:
I don't think this is used in this subroutine

Reply:
Good catch!  Non-used statement removed.